spec/features/new_project_spec.rb

```diff
@@ -28,9 +28,33 @@
     expect(ruby_version_file).to eq "#{RUBY_VERSION}\n"
   end
 
-  it "downloads a .rubocop.yml" do
-    rubocop_file = IO.read("#{project_path}/.rubocop.yml")
+  context "downloaded files" do
+    it "downloads a .rubocop.yml" do
+      rubocop_file = IO.read(File.join(project_path, ".rubocop.yml"))
 
-    expect(rubocop_file).to match %r{Style/StringLiterals}
+      expect(rubocop_file).to match %r{Style/StringLiterals}
+    end
+
+    it "downloads a README.md" do
+      readme_file = IO.read(File.join(project_path, "README.md"))
+
+      expect(readme_file).not_to be_empty
+    end
+  end
+
+  context "copied template files" do
```

Comment:
my goal was that since we have a lot of `invoke` and `build` calls, we should at least ensure we don't delete those calls by mistake. Deleting them wouldn't make the build fail otherwise, but we would end up with stuff missing from the generated Rails app